analytics_data_api/v0/views/learners.py

```diff
@@ -60,6 +60,7 @@ class LearnerView(LastUpdateMixin, CourseViewMixin, generics.RetrieveAPIView):
         Returns metadata and engagement data for the learner in JSON format.
 
             * username: The username of the enrolled learner.
+            * account_url:  URL to learner's account page.
```

Comment:
I'm sort of wondering about whether these URLs should be in here at all now.  If Insights called out to the accounts API directly it could make the calls in parallel and decouple the sources of data a bit.  That way, if for some reason the data api went down, the application could (more) gracefully fail.

Thoughts?